meson.build

```diff
@@ -6,6 +6,8 @@ project('vala-language-server', 'vala', 'c',
   ]
 )
 
+add_global_arguments(['-DG_LOG_DOMAIN="vls"'], language:'c')
```

Comment:
I feel like this is not the right way to do it, because we want `vls-testclient` to have a different log domain, for example. Is there some way to set this for each individual target, and to do it within Vala? (see `src/meson.build` and `test/meson.build`)